src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs

```diff
@@ -39,14 +41,18 @@ public class TestRequestHandler : ITestRequestHandler
     private ICommunicationChannel _channel;
 
     private readonly JobQueue<Action> _jobQueue;
+    private readonly IFileHelper _fileHelper;
     private readonly ManualResetEventSlim _requestSenderConnected;
     private readonly ManualResetEventSlim _testHostManagerFactoryReady;
     private readonly ManualResetEventSlim _sessionCompleted;
     private Action<Message> _onLaunchAdapterProcessWithDebuggerAttachedAckReceived;
     private Action<Message> _onAttachDebuggerAckRecieved;
+    private IPathConverter _pathConverter;
     private Exception _messageProcessingUnrecoverableError;
 
     public TestHostConnectionInfo ConnectionInfo { get; set; }
+    string IDeploymentAwareTestRequestHandler.LocalPath { get; set; }
+    string IDeploymentAwareTestRequestHandler.RemotePath { get; set; }
```

Comment:
Is there any reason to have explicit interface implementation?

Reply:
No strong reason, it just ensures that if we get rid of this interface in the future then all it's properties need to be removed from the type.

Reply:
Yep, only "downside" is that it forces you to make some cast in the code when you want to use it.